### PR TITLE
Pick up the latest integration util changes into c2e-setup

### PR DESCRIPTION
**Pull request by antonia-avramova**
[#208] Pick up the latest integration util changes into c2e-setup

Signed-off-by: Antonia Avramova <antonia.avramova@bosch.io>

### CODE REVIEW

File: integration/c2e-setup/main.go [hunk 1]
```diff
@@ -272,255 +319,109 @@ func performSetUp(deviceResource *resource, resources []*resource) bool {
...
-func restartSuiteConnector() bool {
-	fmt.Println("restarting suite-connector...")
-	cmd := exec.Command(systemctl, restart, suiteConnectorService)
-	stdout, err := cmd.Output()
+func restartService(service string) bool {
+	fmt.Printf("restarting %s...", service)
+	stdout, err := exec.Command(systemctl, restart, service).Output()
 	if err != nil {
-		fmt.Printf("error restarting %s: %v", suiteConnectorService, err)
+		fmt.Printf("error restarting %s: %v", service, err)
```
Comment (gboyvalenkov-bosch) [inline integration/c2e-setup/main.go:421]: Maybe print the output, too. Can it be non-empty, even if there is an error?

File: integration/c2e-setup/main.go [hunk 2]
```diff
@@ -93,15 +94,47 @@ func main() {
...
+		"Path to Local Digital Twins configuration file. "+
+			"If set to the empty string, configuring Local Digital Twins and restarting it will be skipped")
+
+	flag.StringVar(&configLdtFileBackup, "configLdtFileBackup", "/etc/local-digital-twins/configBackup.json",
+		"Path to Local Digital Twins configuration file backup. "+
+			"If set to the empty string, backing up the Local Digital Twins configuration file will be skipped")
+
 	clean := flag.Bool("clean", false, "Clean up test resources")
+	bootstrap = *flag.Bool("bootstrap", false, "Create bootstrapping resources")
+	ldt = *flag.Bool("ldt", false, "Create local-digital-twins resources")
```
Comment (gboyvalenkov-bosch) [inline integration/c2e-setup/main.go:137]: Maybe add validation of flags, i.e check if bootstrap and ldt are both enabled.

File: integration/c2e-setup/main.go [hunk 3]
```diff
@@ -272,255 +319,109 @@ func performSetUp(deviceResource *resource, resources []*resource) bool {
...
+		fmt.Printf(deleteResourcesTemplate, indent, err)
 		ok = false
 	}
 	return ok
 }
 
-func deleteBackupFile() bool {
-	if err := os.Remove(configFileBackup); err != nil {
-		fmt.Printf("unable to delete configuration file backup %s, error: %v", configFileBackup, err)
+func deleteBackupFile(path string) bool {
```
Comment (gboyvalenkov-bosch) [inline integration/c2e-setup/main.go:351]: use something more generic, like deleteFile?

File: integration/c2e-setup/main.go [hunk 4]
```diff
@@ -210,22 +231,34 @@ func assertFlag(value string, name string) {
...
+func getServiceNameConfigAndBackupFile() (string, string, string) {
+	if bootstrap {
+		return suiteBootstrappingService, configBootstrapFile, configBootstrapFileBackup
+	} else if ldt {
+		return ldtService, configLdtFile, configLdtFileBackup
+	}
+	return suiteConnectorService, configConnectorFile, configConnectorFileBackup
+}
+
+func performSetUp(deviceId string, resources []*util.Resource) bool {
```
Comment (gboyvalenkov-bosch) [inline integration/c2e-setup/main.go:249]: rename deviceId to deviceID, check with golint

**Review by gboyvalenkov-bosch — CHANGES_REQUESTED** [review]

**Review by gboyvalenkov-bosch — APPROVED** [review]

**Review by antoniyatrifonova — APPROVED** [review]

**Review by dimitar-dimitrow — APPROVED** [review]

**Pull request merged**